core/Console.php

```diff
@@ -34,11 +34,23 @@ public function __construct()
 
         $this->getDefinition()->addOption($option);
 
+        $option = new InputOption('xhprof',
+            null,
+            InputOption::VALUE_NONE,
+            'Enable profiling with XHProf'
+        );
+
+        $this->getDefinition()->addOption($option);
```

Comment:
Maybe in the summary output (that includes a link to the profile) we can print whether Development mode is enabled, and a note that says nothing is cached and you may want to run w/ it disabled, etc.